ocaml/idl/wire-protocol.tex

```diff
@@ -175,9 +175,10 @@ \subsection{Session Layer}
 The XML-RPC interface is session-based; before you can make arbitrary RPC calls
 you must login and initiate a session. For example:
 \begin{verbatim}
-   session_id    session.login_with_password(string uname, string pwd)
+   session_id    session.login_with_password(string uname, string pwd, string version, string originator)
 \end{verbatim}
 Where {\tt uname} and {\tt password} refer to your username and password
+while {\tt version} and {\tt originator} are optional and refer to the client version and client name 
```

Comment:
IMPORTANT: The version parameter is for the API version, not the client version.